### DISCUSSION TITLE
es/guides/distance-calculation/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#13319] [Docs]
# es/guides/distance-calculation/

Learn how to calculate distances between objects using Ultralytics YOLOv8 for accurate spatial positioning and scene understanding.

https://docs.ultralytics.com/es/guides/distance-calculation/

**Comment by felipeca2** [comment]
It is possible to do it in relation to the jump of a person with respect to the ground? , thanks

**Reply by glenn-jocher** [reply]
Yes, you can calculate the distance of a person's jump relative to the ground using Ultralytics YOLOv8. By tracking the bounding box centroids of the person at different points in the jump, you can measure the vertical distance traveled. Ensure you have a clear view of the person and the ground to accurately set your reference points for measurement. If you need further guidance on setting this up, please refer to our distance calculation guide or feel free to ask here! 🚀
